src/app/component/quiz/quiz.component.scss

```diff
@@ -1,59 +1,105 @@
+.quiz_card {
+  height: auto;
+}
 
-.card{
-    margin: 0 auto; 
-    margin-bottom: 50px;
-    border-radius: 15px;
-    height: auto;
-    
-}
-.card-body{
+.header_txt {
+  position: relative;
+  z-index: 2;
+  height: 70px;
+  padding: 0 15px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  .header_title {
+    font-size: 25px;
+    font-weight: 600;
+    user-select: none;
+  }
+}
 
-    padding: 20px 60px 60px;
-    margin: 8px;
-    display: inline-block;
+.timer {
+  color: #004085;
+  background: #cce5ff;
+  border: 1px solid #b8daff;
+  height: 45px;
+  width: 145px;
+  padding: 0px 8px;
+  border-radius: 5px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  .timer_left_txt {
+    font-size: 17px;
+    font-weight: 600;
+    user-select: none;
+  }
+  .timer_sec {
+    font-size: 18px;
+    font-weight: 500;
+    height: 30px;
+    width: 40px;
+    color: #fff;
     border-radius: 5px;
-    font-size: 20px;
-    box-sizing: border-box;
-    height: auto;
-    width: auto;
-} 
-
-.form-check{
-    margin:20px auto;
-    font-family: Roboto;
-}
-.form-check input[type=radio]{
-    
-    width: 15px;
-    height: 15px;
-}
-.form-check-label{
-    
-    font-size: 16px;
-    margin-bottom: 4px;
+    line-height: 30px;
+    text-align: center;
+    background: #343a40;
+    border: 1px solid #343a40;
+    user-select: none;
+  }
 }
 
-.form-check:hover{
-    font-weight: bold;
-    
-}
-.quiz{
-    color: rgb(0, 0, 0);
-    font-weight:bold;
-    font-size: 20px;  
-    display: flex;
-    justify-content: space-between;
-}
-.quiz:hover{
- color:black;
+.que_txt {
+  padding: 15px 15px;
+  display: inline-block;
+  border-radius: 5px;
+  font-size: 20px;
+  box-sizing: border-box;
+  height: auto;
+  width: auto;
+  .que_data {
+    font-size: 20px;
+    font-weight: 600;
+    user-select: none;
+  }
+}
 
+.option {
+  background: rgb(222, 239, 255);
+  border: 1px solid #84c5fe;
+  border-radius: 5px;
+  padding: 8px 15px;
+  font-size: 17px;
+  cursor: pointer;
+  margin-top: 15px;
+  align-items: center;
+  width: 100%;
+  user-select: none;
 }
-button{
-    font-family: Roboto;
+
+.hightlight-selected {
+  background: rgb(133, 190, 255) !important;
+  border: 1px solid #4a98ff !important;
+  font-weight: 600;
 }
-.data{
-    font-family: Roboto;
+
+.footer_txt {
+  position: relative;
+  z-index: 2;
```

Comment:
i guess z-index:1 will be also same why used z-index 2 ?

Reply:
yes, i checked it and put z-index:1